README.md

```diff
@@ -1,6 +1,6 @@
 # authentik
 
-![Version: 1.0.0-RC7](https://img.shields.io/badge/Version-1.0.0--RC7-informational?style=flat-square) ![AppVersion: 2021.5.2](https://img.shields.io/badge/AppVersion-2021.5.2-informational?style=flat-square)
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0--RC7-informational?style=flat-square) ![AppVersion: 2021.5.2](https://img.shields.io/badge/AppVersion-2021.5.2-informational?style=flat-square)
```

Comment:
I assume this isnt going to merge until 2021.5.3 when the flow ui is fixed?

Reply:
Yeah I'm waiting with 2021.5.3 until I have checked all the forms to make sure I'm not missing anything